process/io/plot_proc.py

```diff
@@ -1326,17 +1326,21 @@ def plot_blanket(axis, mfile_data, scan, colour_scheme) -> None:
         )
         # Plot blanket
         axis.plot(bg_double_null.rs[0], bg_double_null.zs[0], color="black", lw=thin)
-        axis.plot(bg_double_null.rs[1], bg_double_null.zs[1], color="black", lw=thin)
         axis.fill(
             bg_double_null.rs[0],
             bg_double_null.zs[0],
             color=BLANKET_COLOUR[colour_scheme - 1],
         )
-        axis.fill(
-            bg_double_null.rs[1],
-            bg_double_null.zs[1],
-            color=BLANKET_COLOUR[colour_scheme - 1],
-        )
+        if blnkith > 0:
```

Comment:
Should this not be 0.0 as `blnkith` is a `float`

Reply:
Well spotted, done